locales/puppet.pot

```diff
@@ -557,11 +557,16 @@ msgstr ""
 msgid "Applied catalog in %{seconds} seconds"
 msgstr ""
 
+<<<<<<< HEAD
 #: ../lib/puppet/configurer.rb:230
+=======
+#: ../lib/puppet/configurer.rb:224
+>>>>>>> 83e441eba05656a02ddf48723b0ecce27daa15bb
```

Comment:
You'll want to accept `ours` for everything in the `locales` directory, so there shouldn't be any changes to that directory in this PR